code/modules/paperwork/paper_biscuit.dm

```diff
@@ -0,0 +1,125 @@
+		return
+
+	return ..()
+
+/obj/item/folder/biscuit/attack_self(mob/user)
+	add_fingerprint(user)
+	if (!cracked)
+		if (tgui_alert(user, "Do you want to crack it open? You cannot uncrack back.", "Biscuit card", list("Yes", "No")) != "Yes")
+			return
+		cracked = TRUE
+		playsound(get_turf(user), 'sound/effects/wounds/crack1.ogg', 60)
+		icon_state = "[icon_state]_cracked"
+		update_appearance()
+
+	ui_interact(user)
+//Corporate "confidental" biscuit cards
+/obj/item/folder/biscuit/confidental
+	name = "\proper confidental biscuit card"
+	desc = "An confidental biscuit card. In a tasteful blue color with NT logo, looks like a chocolate bar. Has label which says <b>DO NOT DIGEST</b>."
+	icon_state = "paperbiscuit_secret"
+	bg_color = "#355e9f"
+
+/obj/item/folder/biscuit/confidental/spare_id_safe_code
+	name = "\proper spare ID safe code biscuit card"
+	desc = "An biscuit card containing confidental spare ID safe code. In a tasteful blue color with NT logo, looks like a chocolate bar. Has label which says <b>DO NOT DIGEST</b>."
+
+/obj/item/folder/biscuit/confidental/spare_id_safe_code/Initialize(mapload)
+	. = ..()
+	new /obj/item/paper/paperslip/corporate/fluff/spare_id_safe_code(src)
+
+/obj/item/folder/biscuit/confidental/emergency_spare_id_safe_code
+	name = "\proper spare emergency ID safe code biscuit card"
+	desc = "An biscuit card containing <i>not so confidental</i> emergency spare ID safe code. In a tasteful blue color with NT logo, looks like a chocolate bar. Has label which says <b>DO NOT DIGEST</b>."
+
+/obj/item/folder/biscuit/confidental/emergency_spare_id_safe_code/Initialize(mapload)
+	. = ..()
+	new /obj/item/paper/paperslip/corporate/fluff/emergency_spare_id_safe_code(src)
+
+//Biscuits which start not-sealed/cracked initially for the crafting, printing and such
+/obj/item/folder/biscuit/unsealed
+	name = "\proper biscuit card"
+	desc = "An biscuit card. Has label which says <b>DO NOT DIGEST</b>."
+	icon_state = "paperbiscuit_cracked"
+	cracked = TRUE
+	///Was the biscuit already sealed by players? To prevent several tgui alerts
+	var/sealed = FALSE
+	///What is the sprite for when its not cracked? As it starts already cracked, and for re-sealing needs to have a sprite
+	var/not_cracked_icon = "paperbiscuit"
+
+/obj/item/folder/biscuit/unsealed/examine()
+	. = ..()
+	if(!sealed)
+		. += span_notice("This one have not been sealed yet. You many insert anything to seal it by pressing it in hand. Once sealed, the contents are inaccessible until cracked open (irreversible).")
+
+//Asks if you want to seal the biscuit, after you do that it behaves like normal paper biscuit.
+/obj/item/folder/biscuit/unsealed/attack_self(mob/user)
+	add_fingerprint(user)
+	if (!sealed)
+		if (tgui_alert(user, "Do you want to seal it? After sealing the only way to reach the contents is by cracking the biscuit, you cannot re-seal it again after that.", "Biscuit card", list("Yes", "No")) != "Yes")
```

Comment:
```suggestion
		if (tgui_alert(user, "Do you want to seal it? You must crack it open to reach the contents again!", "Biscuit Sealing", list("Yes", "No")) != "Yes")
```

Bit more brief, and the "cracking" menu already says that it's an irreversible thing.